src/me/drton/jmavsim/Simulator.java

```diff
@@ -40,6 +40,7 @@
     private static Port PORT = Port.UDP;
 
     public static boolean   COMMUNICATE_WITH_QGC  = true;   // open UDP port to QGC
+    public static boolean   COMMUNICATE_WITH_SDK  = true;   // open UDP port to SDK
```

Comment:
Shouldn't this be `false` by default?

Reply:
Not from the point of view of an SDK maintainer :D

Reply:
Ha, both should be, yes.

Reply:
See: #100.